.github/workflows/default.yml

```diff
@@ -29,111 +32,110 @@ jobs:
       - name: Docs
         run: make docs
 
+      - name: Toml
+        run: make toml
+
       - name: Porcelain
         run: make porcelain
 
-  build:
-    name: Build
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
+      - name: Test
+        run: make test
 
-      - uses: actions/setup-go@v4
+      - name: Lint
+        uses: golangci/golangci-lint-action@v3
         with:
-          go-version: ^1.20
-        id: go
-
-      - name: Build
-        run: mkdir dist && touch dist/empty && make build
+          version: latest
+          args: --timeout 5m
 
-  test:
-    name: Test
+  check-ui:
+    name: Test & Lint UI
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
 
-      - uses: actions/setup-go@v4
+      - uses: actions/setup-node@v3
         with:
-          go-version: ^1.20
-        id: go
+          node-version: "18"
+          cache: "npm"
+
+      - name: Install tools
+        run: make install-ui
+
+      - name: Lint
+        run: make lint-ui
 
       - name: Test
-        run: mkdir dist && touch dist/empty && make test
+        run: make test-ui
 
-  lint:
-    name: Lint
+      - name: Porcelain
+        run: |
+          test -z "$(git status --porcelain)" || (git status; git diff; false)
```

Comment:
Das wäre auch hier 

    make porcelain

Reply:
Dann würde er `mod tidy` und `go fmt` nochmal ausführen. Das macht er ja in Clean schon. Daher hier nur der Test.